library/global.mli

```diff
@@ -71,6 +72,11 @@ val add_include :
   Entries.module_struct_entry -> bool -> Declarations.inline ->
     Mod_subst.delta_resolver
 
+(** Sections *)
+
+val open_section : poly:bool -> unit
```

Comment:
Please document what `poly` means (I know but people trying to get familiar with the code won't necessarily). I personally would have used a more significant name (like `univpoly`) to avoid the need for documentation but...